contrib/grimshot.1.scd

```diff
@@ -0,0 +1,77 @@
+grimshot(1)
+
+# NAME
+
+grimshot - a helper for screenshots within sway
+
+# DESCRIPTION
+
+grimshot [--notify] (copy|save) [TARGET] [FILE]
+grimshot check
+grimshot usage
+
+*--notify*
+	Show notifications to the user that a screenshot has been taken.
+
+*save*
+	Save the screenshot into a regular file. Grimshot will write images
+	files to *$XDG_SCREENSHOTS_DIR* if this is set (or defined in
+	`user-dirs.dir`), or otherwise fall back to `$XDG_PICTURES_DIR`
+
+*copy*
+	Copy the screenshot data (as image/png) into the clipboard.
+
+# SYNOPSIS
+
+Grimshot is an easy to use screenshot tool for sway. It relies on grim, slurp
+and jq to do the heavy lifting, and mostly provides an easy to use interface.
+
+A recommended usage pattern is to just bind this to Super+P via sway:
+
+```
+# Screenshots:
+# Cmd+P: Current window
+# Cmd+Shift+p: Select region
+# Cmd+Alt+p Current output
+# Cmd+Ctrl+p Select a window
+
+bindsym Mod4+p       exec grimshot active
+bindsym Mod4+Shift+p exec grimshot region
+bindsym Mod4+Mod1+p  exec grimshot output
+bindsym Mod4+Ctrl+p  exec grimshot window
+```
+
+# TARGETS
+
+grimshot can capture the following named targets:
+
+_active_
+	Captures the currently active window.
```

Comment:
It seems like the actual script uses `win` target name, not `active`.

Reply:
Good catch, fixed.